cmd/manager.go

```diff
@@ -36,7 +36,7 @@ And it also waits for reports from support bundle agents. The reports contain:
 
 func init() {
 	rootCmd.AddCommand(managerCmd)
-	managerCmd.PersistentFlags().StringVar(&sbm.HarvesterNamespace, "harvester-namespace", os.Getenv("HARVESTER_NAMESPACE"), "The Harvester namespace")
+	managerCmd.PersistentFlags().StringVar(&sbm.NamespaceList, "namespaces", os.Getenv("NAMESPACES"), "List of namespaces delimited by :")
```

Comment:
Can we use `,` instead of `:`? Haven't seen the usage of `:` as the delimiter much.

Reply:
I was thinking of the unix PATH, but I updated it to use comma.